pkgs/by-name/pd/pdqsort/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenvNoCC,
+  stdenv, # for tests
+  fetchFromGitHub,
+}:
+
+stdenvNoCC.mkDerivation (fa: {
+  pname = "pdqsort";
+  version = "unstable-2021-03-14";
+
+  src = fetchFromGitHub {
+    owner = "orlp";
+    repo = "pdqsort";
+    rev = "b1ef26a55cdb60d236a5cb199c4234c704f46726";
+    sha256 = "sha256-xn3Jjn/jxJBckpg1Tx3HHVAWYPVTFMiDFiYgB2WX7Sc=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/lib/pkgconfig $out/include
+    cp -r *.h $out/include/
+  '';
+
+  # The benchmark takes too long to run as a regular checkPhase here.
```

Comment:
how long does it take ?

Reply:
4-5 minutes

Reply:
it's ok to include part of checkPhase, but I'll leave this up to you